form.css

```diff
@@ -2,7 +2,7 @@
 
 :root{
     --color-white:#f3f3f3;
-    --color-darkblue:#796ec8;
+    --color-darkblue:#3824cf;
```

Comment:
Use the correct background color; color-white:#f3f3f3;

Reply:
- [ ] Kindly always include an explanation, or a "why", for each requested change, this will make your comments more useful and clear rg why using the correct background color is a good thing.